### ISSUE TITLE
Arko'narin escort, caged night elf NPC in Felwood Jaedenar cave

### ISSUE THREAD

**Issue by aps-kar** [LightsHope/issues#1368]
I found out how she bugs out, because it just happened to me.
I got mind controlled from one of the satyrs close to her cage and started attacking her, after the mind control wore off, she ran back to her starting position inside the cage.
The quest wasn't failed, but she is now stuck and after abandoning the quest, she won't re-issue it to me, or any other player for that matter.

**Comment by aps-kar** [comment]
From experience, this is only fixed by server restart.

**Comment by Dugrem** [comment]
I was unable to turn the key in to her at all. When the key drops you can accept the quest but she is unavailable for turn in. She just stays in the cage with no indicators. (No question mark)

**Comment by aps-kar** [comment]
Yes, that's because you found her in the bugged/stuck stage I described above. I had the same issue yesterday, but when I came back to check on her today after the server crash and restart, I was able to turn the quest in to her and start her escort.
